messaging/src/main/java/org/axonframework/queryhandling/QuerySubscription.java

```diff
@@ -32,7 +32,7 @@
  * @author Steven van Beelen
  * @since 3.2
  */
-class QuerySubscription<R> {
+public class QuerySubscription<R> {
```

Comment:
Why did we need this class to be `public` for the multi-tenancy changes?

Reply:
access to this class is needed in MultiTenantQueryBus